content/zh/boilerplates/verify-crds.md

```diff
@@ -1,4 +1,4 @@
-Wait for all Istio CRDs to be created:
+等待所有的 Istio CRDs 创建完成：
```

Comment:
CRDs -> CRD

Reply:
@idefav 	update file according to review

Reply:
updated